### PR TITLE
add One Creation Source documentation

### PR DESCRIPTION
**Pull request by yanghcheung**
### Proposed changes

add One Creation documentation

### Merge timing
ASAP once approved

### Related issues (optional)

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/onecreation/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: 'One Creation Source'
+---
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/onecreation/index.md:3]:
```suggestion
---
title: 'One Creation Source'
id: IB9M67ZWaA
---
```

Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/onecreation/index.md:3]:
```suggestion
---
title: 'One Creation Source'
id: IB9M67ZWaA
---
```

File: src/connections/sources/catalog/cloud-apps/onecreation/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+This source is maintained by One Creation. For any issues with the source, [contact their Support team](mailto:support@one-creation.com).
+
+## Getting started
+
+1. From your workspace's Sources catalog page{:target="_blank”} click **Add Source**.
+2. Search for "One Creation" in the Sources Catalog, select One Creation, and click Add Source.
+3. On the next screen, give the Source a name configure any other settings.
+
+    The name is used as a label in the Segment app, and Segment creates a related schema name in your warehouse. The name can be anything, but we recommend using something that reflects the source itself and distinguishes amongst your environments (eg. SourceName_Prod, SourceName_Staging, SourceName_Dev).
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/onecreation/index.md:17]:
```suggestion
- The name is used as a label in the Segment app, and Segment creates a related schema name in your warehouse. The name can be anything, but we recommend using something that reflects the source itself and distinguishes amongst your environments (eg. OneCreation_Prod, OneCreation_Staging, OneCreation_Dev).
```

File: src/connections/sources/catalog/cloud-apps/onecreation/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+4. Click Add Source to save your settings.
+5. Copy the Write key from the Segment UI.
+6. Log in to your [One Creation account](https://app.one-creation.com/admin/integrations?app=segment) - navigate to Settings > Integrations > Segment and paste the key to connect.
+
+## Stream
+
+One Creation uses our stream Source component to send Segment event data. It uses a server-side (select from `track`, `identify`) method(s) to send data to Segment. These events are then available in any destination that accepts server-side events, and available in a schema in your data warehouse, so you can query using SQL.
+
+One Creation assigns a unique ID to each user. This ID is passed to Segment as theuserID. The user's email address is included as a trait.
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/onecreation/index.md:27]:
```suggestion
One Creation assigns a unique ID to each user. This ID is passed to Segment as the `userId`. The user's email address is included as a trait.
```

**Review by tcgilbert — DISMISSED** [review]
In addition to the requested changes, please change the folder name to be the same as the integration slug: 

- `onecreation` --> `one-creation`

**Review by pwseg — DISMISSED** [review]

**Review by pwseg — APPROVED** [review]

**Pull request merged**